Mapify/SceneInitializers/GameContent/WorldMapSetup.cs

```diff
@@ -80,36 +80,44 @@ public static void UpdateMap(Transform transform)
                 modifiedMaterial = true;
             }
 
-            Transform names = transform.FindChildByName("Names");
-            if (names == null)
+            ShowNamesOnMap(mapTransform, isStationOfficeMap);
+        }
+
+        private static void ShowNamesOnMap(Transform mapTransform, bool isStationOfficeMap)
```

Comment:
What's the purpose of the isStationOfficeMap parameter? Please don't include seemingly unrelated changes to the PR without at least explaining the purpose in the description. It would save me time trying to figure out what all you've done and why.

Reply:
My bad. TBH, I don't really remember why I did that but it seems incorrect. Removed.